Corona-Warn-App/src/main/assets/privacy_tr.html

```diff
@@ -1,10 +1,112 @@
+    birlikte, yeni COVID sertifikalarını uygulamaya kaydedemezsiniz ve herhangi bir COVID sertifika
+    yenileme bildirimi almazsınız. Kayıtlı COVID sertifikalarını ve temas günlüğünü kullanmaya devam
+    etmek istiyorsanız, lütfen bu bölümün sonundaki notlara bakın.
+    Uyku Modunu etkinleştirdiğinizde Corona-Warn-App, karşılaşma verilerini göndermek ve
+    karşılaşmaları kaydetmek için kullanılan işletim sisteminizin COVID-19 bildirim sistemini devre
+    dışı bırakacaktır (iPhone: "Maruz Kalma Bildirimleri"; Android akıllı telefonlar: "COVID-19
+    Bildirimleri"). COVID-19 bildirim sistemini istediğiniz zaman işletim sisteminizin ayarlarından
+    kendiniz de devre dışı bırakabilirsiniz. İşletim sisteminiz 14 gün sonra akıllı telefonunuzda
+    depolanan maruz kalma verilerini otomatik olarak siler. Daha önce silmek isterseniz, maruz kalma
+    verilerini işletim sisteminizin ayarlarından manuel olarak da silebilir veya Corona-Warn-App
+    uygulamasından kaldırabilirsiniz.
+    Uluslararası uyarıya katılan ülkelerin değişim sunucuları çoktan kapatılmıştır. Uluslararası
+    uyarılarla bağlantılı olarak işlenen tüm kişisel veriler silinmiştir.
+    01 Haziran 2023 tarihinden itibaren Corona-Warn-App artık geliştirilmeyecektir.
+    <u>Dijital COVID sertifikaları için işlevlerin kullanımı ve 1 Mayıs 2023'ten itibaren temas
+        günlüğü
+        hakkında notlar:</u>
+</p>
+<ul>
+    <li>
+        Dışa aktarma işlevlerini kullanarak kayıtlı COVID dijital sertifikalarınızı ve temas günlüğü
+        girişlerinizi yedeklemeye devam edebilirsiniz. Dijital bir COVID sertifikasını kanıt olarak
+        göstermek istemeniz durumunda, lütfen sertifikaların güvenli bir şekilde kullanılmasıyla
+        ilgili olarak aşağıda ekli olan gizlilik politikasındaki talimatları izleyin (madde Hata!
+        Başvuru kaynağı bulunamadı.) ve sertifikaları yalnızca uygun bir doğrulama uygulamasıyla
+        taratın.
+    </li>
+    <li>
+        Akıllı telefonunuzda depolanan temas günlüğündeki girişleri ve depolanan dijital COVID
+        sertifikalarını tamamen silmek istiyorsanız, lütfen Corona-Warn-App uygulamasını kaldırın.
+    </li>
+    <li>
+        Corona-Warn-App uygulamasının önümüzdeki birkaç hafta içinde uygulama mağazalarından
+        kaldırılacağını lütfen unutmayın. Bu durumda yeni bir kurulum artık mümkün olmayacaktır.
+    </li>
+</ul>
+<p>
+    <strong>2. Verileriniz kime aktarılacak?</strong>
+</p>
+<p>
+    1 Mayıs 2023 tarihinden itibaren RKI, Corona-Warn-App uygulamasının çalışması için artık kişisel
+    verileri işlemeyecektir. Boş modda, Corona-Warn-App artık RKI'nin sunucu sistemine veya diğer
+    alıcılara veri aktarmayacaktır. Corona-Warn-App yalnızca akıllı telefonunuzdaki verilerinizi
+    (temas günlüğündeki girişler ve kayıtlı COVID sertifikaları) işler.
+</p>
+<p>
+    <strong>3. Wie können Sie Ihr Einverständnis zurücknehmen?</strong>

```

Comment:
here are some german heading - translation should be like this (checked with a turkish speaking colleague) 

3 Onayınızı nasıl geri çekebilirsiniz?

Reply:
Ok